src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -10,10 +10,12 @@
         {{workItem.id}}
       </div>
     </div>
-    <div class="form-group">
+    <div class="form-group" [ngClass]="{'has-error': !workItem.fields['system.title']}">
```

Comment:
It looks like this is fix from PR (#119). If you need this fix in here for the PR to work then the commits need to be merged. If you don't need it then it should be removed or at least a PR merge order should be listed. (something like please merge 119 before this one).